<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRelationEditorConfigWidgetBase</class>
 <widget class="QWidget" name="QgsRelationEditorConfigWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attribute Widget Relation Edit Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="mShowFirstFeature">
     <property name="toolTip">
      <string>Unchecking this can lead to faster loading time of attribute forms and avoid unnecessary queries.</string>
     </property>
     <property name="text">
      <string>Automatically select first child feature and show attribute form</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="mButtonsVisibility">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>4</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Buttons and Context Menu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="mRelationShowLinkCheckBox">
        <property name="text">
         <string>Link child feature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mRelationShowUnlinkCheckBox">
        <property name="text">
         <string>Unlink child feature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mRelationShowSaveChildEditsCheckBox">
        <property name="text">
         <string>Save child layer edits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mRelationShowAddChildCheckBox">
        <property name="text">
         <string>Add child feature</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="mAllowAddChildFeatureWithNoGeometry">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, it is possible to add a feature without geometry to a spatial layer. Two &amp;quot;Add child feature&amp;quot; buttons are showed: with and without geometry.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Allow add child feature with no geometry (for spatial layer)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="mRelationShowDuplicateChildFeatureCheckBox">
        <property name="text">
         <string>Duplicate child feature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mRelationDeleteChildFeatureCheckBox">
        <property name="text">
         <string>Delete child feature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mRelationShowZoomToFeatureCheckBox">
        <property name="text">
         <string>Zoom to child feature</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="filterExpressionLabel">
          <property name="text">
           <string>Filter expression</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mEditExpression">
          <property name="icon">
           <iconset resource="../../images/images.qrc">
            <normaloff>:/images/themes/default/mIconExpression.svg</normaloff>:/images/themes/default/mIconExpression.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="mFilterExpression"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
